addons/sale_stock/models/sale_order.py

```diff
@@ -170,28 +186,63 @@ def _onchange_product_uom_qty(self):
         return {}
 
     @api.multi
-    def _prepare_order_line_procurement(self, group_id=False):
-        vals = super(SaleOrderLine, self)._prepare_order_line_procurement(group_id=group_id)
+    def _prepare_procurement_values(self, group_id=False):
```

Comment:
A docstring about what this method is intended would be good (as it's not the typical dictionary for creating a record).